### PR TITLE
Finish PutMarksView

### PR DESCRIPTION
**Pull request by mike-kob**

### CODE REVIEW

File: Models/Mark.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using BD_oneLove.Tools;
+
+namespace BD_oneLove.Models
+{
+    internal class Mark : BaseViewModel
+    {
+        public Mark()
```
Comment (dashayaskova) [inline Models/Mark.cs]: why dont you use default constructor?

Comment (dashayaskova) [inline Models/Mark.cs]: why dont you use default constructor?

**Review by dashayaskova — CHANGES_REQUESTED** [review]
Are your views refreshing every time when you open them?

**Comment by mike-kob** [comment]
> Are your views refreshing every time when you open them?

Nope

**Review by dashayaskova — APPROVED** [review]

**Pull request merged**